### PR TITLE
DEV: Add Deutsch translations

### PR DESCRIPTION
**Pull request by Justman100**

### CODE REVIEW

**Comment by Justman100** [comment]
@Arkshine

File: locales/de.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+en:
```
Comment (Arkshine) [inline locales/de.yml]: I believe this should be `de` here.

**Review by Arkshine — CHANGES_REQUESTED** [review]
Thanks!

File: locales/de.yml (hunk 1 above)
Comment (Justman100) [reply locales/de.yml]: Fixed

**Review by Arkshine — APPROVED** [review]

**Pull request merged**